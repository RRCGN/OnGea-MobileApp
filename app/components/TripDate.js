import React from 'react'
import { View, Text, StyleSheet } from 'react-native'

const styles = StyleSheet.create({
  typeText: {
    fontSize: 12,
    color: 'rgba(0,0,0,0.54)'
  },
  dateText: {
    fontSize: 16,
    fontWeight: '500',
    color: 'rgba(0,0,0,0.87)'
  }
})

const TripDate = ({ typeText, dateText }) => {
  return (
    <View>
      <Text style={styles.typeText}>{typeText}</Text>
      <Text style={styles.dateText}>{dateText}</Text>
    </View>
  )
}

export default TripDate
