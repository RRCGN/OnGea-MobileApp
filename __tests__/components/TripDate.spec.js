import 'react-native'
import React from 'react'
import TripDate from '../../app/components/TripDate'

import renderer from 'react-test-renderer'

describe('<TripDate />', () => {
  test('renders correctly', () => {
    const tree = renderer.create(
      <TripDate type="vom" date="2017-04-10T02:00:00+02:00" />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
