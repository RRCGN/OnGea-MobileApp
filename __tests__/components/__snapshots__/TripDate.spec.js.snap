exports[`<TripDate /> renders correctly 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "rgba(0,0,0,0.54)",
        "fontSize": 12,
      }
    }>
    vom
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "rgba(0,0,0,0.87)",
        "fontSize": 16,
        "fontWeight": "500",
      }
    }>
    10.04.2017
  </Text>
</View>
`;
